spec/dummy/app/themes/default/views/layouts/thredded.html.erb

```diff
@@ -9,7 +9,16 @@
 
   <%= content_tag(:body, id: (yield :thredded_page_id), class: 'layout-application') do %>
     <%= render 'thredded/shared/icons_svg' %>
-    <%= yield %>
+
+    <div class="main-container">
+      <header class="navigation" role="banner">
+        <%#= render 'thredded/shared/top_nav' %>
+        <%#= render 'thredded/shared/topic_nav' %>
```

Comment:
Ah ok, @kylefiedler, I think I get you now. If you want to put shared partials in the right place they would end up in `spec/dummy/app/themes/default/views/thredded/shared`.